docs/docusaurus/docs/reference/learn/integrations/dbt_tutorial.md

```diff
@@ -378,7 +378,7 @@ Add Postgres credentials:
     - Connection id: postgres
     - Connection Type: Postgres
     - Host: database
-    - Schema: postgres
+    - Database: postgres
```

Comment:
Note for reviewer: This appears to have changed in the airflow UI. Here's a side by side of airflow vs the screenshot in the docs. (I don't think the mismatch is severe enough to warrant the effort to update the screenshot, but I do think it's worth updating the prose)